src/main/java/org/jvnet/winp/Native.java

```diff
@@ -70,7 +71,20 @@
     private static final String DLL_TARGET = "winp.folder.preferred";
     private static final String UNPACK_DLL_TO_PARENT_DIR = "winp.unpack.dll.to.parent.dir";
 
-    private static String ctrlCExePath;
+    private static final LazyValue<String> ctrlCExePathLazyValue = new LazyValue<String>(new Callable<String>() {
+        public String call() {
+            File exeFile = load();
+            return exeFile == null ? null : exeFile.getPath();
+        }
+    });
+
+    static {
+        try {
+            ctrlCExePathLazyValue.getValue(); // load winp library
+        } catch (Throwable e) {
```

Comment:
maybe `Exception`? I would prefer `Error`s to be fatal unless there is a clear reason to suppress them

Reply:
IIUC, in case of `LinkageError` thrown, that would make org.jvnet.winp.Native uninitialized (#56).